tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -59,6 +61,61 @@ def test_pcluster_configure(
     clusters_factory(config_path)
 
 
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
+@pytest.mark.schedulers(["awsbatch", "slurm", "sge"])
+# Do not run on ARM + Batch
+# pcluster configure always picks optimal and Batch does not support ARM for optimal for now
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "*", "awsbatch")
```

Comment:
We probably only need to run this test once with a particular os/scheduler/instance combination, because what we are testing do not depend on these 3 dimensions